tdrs-backend/tdpservice/data_files/models.py

```diff
@@ -149,9 +149,14 @@ class Meta:
     @property
     def filename(self):
         """Return the correct filename for this data file."""
+        # This logic is temporary until the section choices for tribe and SSP are available
         if str(self.stt.type).lower() == 'tribe':
             return self.stt.filenames.get(
-                ('Tribal ' if 'Tribal' not in self.section else '') + self.section,
+                ('Tribal ' if 'Tribal' not in self.section else '') + self.section,  # This is temporary logic
+                None)
+        elif self.stt.ssp:  # Currently, this doesn't exists and has to be added
+            return self.stt.filenames.get(
+                ('SSP ' if 'SSP' not in self.section else '') + self.section,  # This is temporary logic
```

Comment:
Since only DRF/API is used to create data_file, and we are adding SSP to the section in serializer, then there is no need to add the logic in the model.

nit: can add the same logic for 'Tribal' in serializer instead of create method in model